test/v2/services/MovementsServiceSpec.scala

```diff
@@ -122,9 +113,9 @@ class MovementsServiceSpec
     "on a successful submission, should return a Right" in {
       implicit val hc: HeaderCarrier = HeaderCarrier()
       when(mockConnector.postMovement(EORINumber(any[String]), any(), any())(eqTo(hc), any[ExecutionContext]))
-        .thenReturn(Future.successful(MovementResponse(MovementId("ABC"), None)))
+        .thenReturn(Future.successful(MovementResponse(MovementId("ABC"), MessageId("123"))))
```

Comment:
movementID and messageId should be 16 digit for version 2 api. Good if we can test with right data otherwise please ignore

Reply:
it's a test data, it should be fine

Reply:
I would rather test data reflect real world cases as much as possible when it affects the test -- but I'm as guilty as others for doing this similar thing.

Would rather it was at least a 16 character hex, and maybe it should use the generators, but considering what this test is, I'd not get hung up on that.

Reply:
anyways, updated to use 16 digit to reflect real world cases